docs/src/apps/backwards-compatibility.md

```diff
@@ -0,0 +1,110 @@
+---
+title: Backwards Compatibility Policy
+---
+
+As the Solana developer ecosystem grows, so does the need for clear expectations around
+breaking API and behavior changes affecting applications and tooling built for Solana.
+In a perfect world, Solana development could continue at a very fast pace without ever
+causing issues for existing developers. However, some compromises will need to be made
+and so this document attempts to clarify and codify the process for new releases.
+
+### Expectations
+
+- Solana APIs, SDKs, and CLI tooling are tied to validator software releases (with a few
```

Comment:
The monorepo release is also morphing into the program SDK release as well, it's not really a validator software release anymore.  The "Solana software release"?